recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 
 build:
   number: 0
-  skip: True  # [py3k]
+  skip: True  # [osx and py3k]
```

Comment:
The re-rendering looks OK but note that you are missing Python 3.6 b/c one or more dependencies are missing for Python 3.6. To fix that you must find which ones and re-render there too.